CHANGELOG.md

```diff
@@ -55,6 +60,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - Generic template (`InfluxDBTemplate`) to interact with InfluxDB
 
 [Unreleased]: https://github.com/miwurster/spring-data-influxdb/compare/spring-data-influxdb-1.8...HEAD
+[2.0]: https://github.com/miwurster/spring-data-influxdb/compare/spring-data-influxdb-1.8...spring-data-influxdb-2.0
```

Comment:
Not required. I'll do this after the release has been published.

Reply:
it is fixed.